tests/unittests/server/utils/jsonpatch/test_jsonpatch.py

```
@@ -173,3 +173,98 @@ def test_ok_multi(transform_fn, patches):
+
+    def test_apply_does_not_raise_for_multiple_operations(self):
+        """apply() does not raise RootDeleteError for multiple operations."""
+        target = {"label": "cat"}
+        operations = [
+            {"op": "remove", "path": "/"},
+            {"op": "add", "path": "/label", "value": "dog"},
+        ]
+
+        result, errors = jsonpatch.apply(target, operations)
+        assert errors == []
+        assert "label" in result and result["label"] == "dog"
```

Ruff (0.14.13)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)